pymc/model.py

```diff
@@ -2035,10 +2035,49 @@ def Deterministic(name, var, model=None, dims=None):
 def Potential(name, var, model=None):
     """Add an arbitrary factor potential to the model likelihood
 
+      The Potential function is used to add arbitrary factors (such as constraints or other likelihood components) to adjust the probability density of the model.
+
+      Have a look at the following example:
+
+     .. code:: python
+
+        with pm.Model() as model:
+          x = pm.Normal('x', mu=0, sigma=1)
+          y = pm.Normal('y', mu=x, sigma=1, observed=data)
+          constraint = x>= 0
+          potential = pm.Potential('x_constraint', -pm.math.log(pm.math.switch(constraint, 1, 1e-30))
```

Comment:
I don't think it should be negated

```suggestion
          potential = pm.Potential('x_constraint', pm.math.log(pm.math.switch(constraint, 1, 1e-30))
```